### PR TITLE
Change top contributors to a drop-down list on results page [SIDASH-99]

### PR DESCRIPTION
**Pull request by Rytiggy**
Change top contributors to a drop-down list of all contributors 

*** Do not merge until 06/23/17, as there is a potential bug that needs to be checked again after Share is under less stress***

### CODE REVIEW

File: app/components/dropdown-widget/component.js [hunk 1]
```diff
@@ -55,18 +54,23 @@ export default Ember.Component.extend({
         Ember.$(document).off('click', clickHandler);
     },
     processData (data) {
+        try{
```
Comment (samanehsan) [inline app/components/dropdown-widget/component.js]: Please remove this try/catch code leftover from debugging

**Comment by samanehsan** [comment]
@Rytiggy what is the potential bug that needed to get checked?

**Comment by samanehsan** [comment]
The current selection should show the contributor name instead of the id. @caneruguz may have already fixed this for the funders widget though so you should check in with him!  

screen shot 2017-06-22 at 11 22 13 am

**Comment by Rytiggy** [comment]
The Potential bug that needs check is with share and a 504 error that gets returned. because of this, the contributor list doesn't load in when the 504 error is returned. I have talked with Share and they said that it is possible that it is happening currently because Share is under stress and should be fine in a few days.

**Comment by Rytiggy** [comment]
Currently, the funders widget does not display the name of the funders @samanehsan 
screen shot 2017-06-22 at 12 05 14 pm

**Comment by samanehsan** [comment]
Thanks for clarifying that @Rytiggy! @caneruguz fixed the displaying issue in https://github.com/cos-labs/share-analytics/pull/193/commits/ede78928ce37689ef567934468f19ab9c7fbdc3c, but that hasn't been merged yet. So I would suggest whoever's PR gets merged second should check that it also works for contributors!

**Comment by Rytiggy** [comment]
@samanehsan It turns out that the query is just too complex. It works for the most part currently but needs more testing before its merged.

**Comment by samanehsan** [comment]
@Rytiggy can you make the currently selected contributor display the name instead of the contributor ID? You can take a look at how the funders widget handles this, since that fix is now on develop.

**Comment by Rytiggy** [comment]
@samanehsan yes I can

File: app/components/dropdown-widget/component.js [hunk 2]
```diff
@@ -29,12 +29,14 @@ export default Ember.Component.extend({
         // Show the selected parameter
         let queryParams = this.get('parameters');
         var facet = this.get("item.facetDashParameter");
-
+        console.log(queryParams[facet])
```
Comment (birdbrained) [inline app/components/dropdown-widget/component.js]: Extraneous `console.log`

File: app/components/dropdown-widget/component.js [hunk 3]
```diff
@@ -47,6 +49,7 @@ export default Ember.Component.extend({
         }
     },
     fetchAgentDetails: async function(agentList) {
+        console.log('agentList' , agentList);
```
Comment (birdbrained) [inline app/components/dropdown-widget/component.js]: console.log

**Comment by samanehsan** [comment]
@Rytiggy would you mind making these minor changes? Let's get this merged since it's been hanging out in code-review for a while. Thanks!

**Comment by NyanHelsing** [comment]
Contributors and funders have been removed from these pages.

**Pull request merged**